src/main/java/org/openstreetmap/josm/plugins/pt_assistant/utils/HelpUtils.java

```diff
@@ -0,0 +1,81 @@
+package org.openstreetmap.josm.plugins.pt_assistant.utils;
+
+import static org.openstreetmap.josm.tools.I18n.tr;
+
+import org.openstreetmap.josm.gui.MainApplication;
+import org.openstreetmap.josm.gui.MapFrame;
+import org.openstreetmap.josm.gui.widgets.JMultilineLabel;
+import org.openstreetmap.josm.tools.GBC;
+import org.openstreetmap.josm.tools.ImageProvider;
+import org.openstreetmap.josm.data.preferences.BooleanProperty;
+
+import javax.swing.*;
+import javax.swing.border.CompoundBorder;
+import javax.swing.border.EmptyBorder;
+import javax.swing.border.EtchedBorder;
+import java.awt.Color;
+import java.awt.Cursor;
+import java.awt.Font;
+import java.awt.GridBagLayout;
+
+/**
+ * Help messages that are displayed on the bottom of the map
+ *
+ * @author sudhanshu2
+ */
+public class HelpUtils extends JPanel {
+
+    private static final HelpUtils ClassInstance = new HelpUtils();
```

Comment:
Variable names should always have a lower case letter in the beginning (classInstance).

Since this is a constant, you can write it all upper case: CLASS_INSTANCE